app/views/layouts/application.html.erb

```diff
@@ -10,8 +10,12 @@
   <body>
     <%= render "layouts/header" %>
     <div class="container">
+      <% flash.each do |message_type, message| %>
+        <div class="alert alert-<%= message_type %>"><%= message %></div>
+      <% end %>
       <%= yield %>
     </div>
     <%= render "layouts/footer" %>
+    <%= debug(params) if Rails.env.development? %>
```

Comment:
Can you tell me what is the purpose of using this line and how you will be benefited from this code snippet?

Reply:
Because we don't use test env in this app so if Rails.env.development? is true, we restrict the debug display to development only. Inserting the debug information into tests will not have error, but it probably wouldn’t do any good,

Reply:
@maiq721 You can see the information about the params by using this, which will help you to debug and understand the code easily and it only executes this code snippet if the the environment is development(Because in the production you don't need this)

Reply:
remove `()`